<!DOCTYPE html>
<!--[if IE 8]><html class="no-js lt-ie9" lang="en" > <![endif]-->
<!--[if gt IE 8]><!--> <html class="no-js" lang="en" > <!--<![endif]-->
<head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  
  <meta name="author" content="Amy J Baldwin">
  <link rel="canonical" href="https://volac-genomics.github.io/Tool3_docs/faq_3.html">
  <link rel="shortcut icon" href="img/favicon.ico">
  <title>FAQs - Tool 3</title>
  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lato:400,700|Roboto+Slab:400,700|Inconsolata:400,700" />

  <link rel="stylesheet" href="css/theme.css" />
  <link rel="stylesheet" href="css/theme_extra.css" />
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.5.0/styles/github.min.css" />
  
  <script>
    // Current page data
    var mkdocs_page_name = "FAQs";
    var mkdocs_page_input_path = "faq_3.md";
    var mkdocs_page_url = "/Tool3_docs/faq_3.html";
  </script>
  
  <script src="js/jquery-2.1.1.min.js" defer></script>
  <script src="js/modernizr-2.8.3.min.js" defer></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.5.0/highlight.min.js"></script>
  <script>hljs.initHighlightingOnLoad();</script> 
</head>

<body class="wy-body-for-nav" role="document">

  <div class="wy-grid-for-nav">

    
    <nav data-toggle="wy-nav-shift" class="wy-nav-side stickynav">
    <div class="wy-side-scroll">
      <div class="wy-side-nav-search">
        <a href="index.html" class="icon icon-home"> Tool 3</a>
        <div role="search">
  <form id ="rtd-search-form" class="wy-form" action="./search.html" method="get">
      <input type="text" name="q" placeholder="Search docs" title="Type search term here" />
  </form>
</div>
      </div>

      <div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="main navigation">
                <ul>
                    <li class="toctree-l1"><a class="reference internal" href="index.html">Introduction</a>
                    </li>
                </ul>
                <ul>
                    <li class="toctree-l1"><a class="reference internal" href="quick_3.html">Quick start</a>
                    </li>
                </ul>
                <p class="caption"><span class="caption-text">Running Tool 3</span></p>
                <ul>
                    <li class="toctree-l1"><a class="reference internal" href="access_3.html">Accessing the server</a>
                    </li>
                    <li class="toctree-l1"><a class="reference internal" href="loading_3.html">Loading query genomes</a>
                    </li>
                    <li class="toctree-l1"><a class="reference internal" href="running_3.html">Initiating Tool 3</a>
                    </li>
                    <li class="toctree-l1"><a class="reference internal" href="options_3.html">Customisation</a>
                    </li>
                    <li class="toctree-l1"><a class="reference internal" href="unloading_3.html">Downloading outputs</a>
                    </li>
                </ul>
                <p class="caption"><span class="caption-text">Further Information</span></p>
                <ul class="current">
                    <li class="toctree-l1 current"><a class="reference internal current" href="faq_3.html">FAQs</a>
    <ul class="current">
    </ul>
                    </li>
                    <li class="toctree-l1"><a class="reference internal" href="glossary_3.html">Glossary</a>
                    </li>
                    <li class="toctree-l1"><a class="reference internal" href="troubleshooting_3.html">Troubleshooting</a>
                    </li>
                </ul>
      </div>
    </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap">

      
      <nav class="wy-nav-top" role="navigation" aria-label="top navigation">
        <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
        <a href="index.html">Tool 3</a>
      </nav>

      
      <div class="wy-nav-content">
        <div class="rst-content">
          <div role="navigation" aria-label="breadcrumbs navigation">
  <ul class="wy-breadcrumbs">
    <li><a href="index.html">Docs</a> &raquo;</li>
    
      
        
          <li>Further Information &raquo;</li>
        
      
    
    <li>FAQs</li>
    <li class="wy-breadcrumbs-aside">
      
        <a href="https://github.com/volac-genomics/Tool3_docs/edit/master/docs/faq_3.md"
          class="icon icon-github"> Edit on GitHub</a>
      
    </li>
  </ul>
  
  <hr/>
</div>

          <div role="main">
            <div class="section">
              
                <h2 id="which-strains-can-you-compare">Which strains can you compare?</h2>
<p>You can compare any bacterial strains. </p>
<p>Practically, the strains you have available to use are those strains in the Tool 3 libraries. These include the strains that have been sequenced as part of this project, and also a selection of reference strains.</p>
<h2 id="how-many-strains-can-you-compare">How many strains can you compare?</h2>
<p>You can compare two or more strains. </p>
<p>It is recommended that you limit strains to only those that are essential for your query. The more genomes you compare the longer Tool 3 will take to run and the larger and more complex the output files will be. Although the application can compare more than 100 strains, comparing 12 strains would be a lot.</p>
<h2 id="what-is-the-best-way-to-select-strains">What is the best way to select strains?</h2>
<p>This will largely depend on what question you want to answer. Tool 3 will give you the best results when comparing closely related strains rather than very diverse strains. It may be useful to look at phylogenetic trees when selecting strains. </p>
<h2 id="what-information-does-tool-3-need">What information does Tool 3 need?</h2>
<p>Tool 3 compares annotated genome files. Tool 3 is pre-populated with two libraries: one library contains the annotated files for every sequenced isolate, and the other library contains the annotated files for various reference genomes. </p>
<p>You will find these files in folders <code>LIBRARY_query</code> and <code>LIBRARY_reference</code>. The file type is <code>.gff</code>, which can be generated by <code>prokka</code>. </p>
<h2 id="how-do-you-run-tool-3">How do you run Tool 3?</h2>
<p>A step-by-step guide can be found in the following sections.</p>
<p>Briefly, once you have chosen which strains you wish to compare, copy the relevant annotation files (<code>.gff</code>) from the libraries to the input directory (<code>input_tool3</code>), set up a stable connection to the server and run the <code>tool_3.nf</code> script. Your output files will be deposited in the output directory (<code>output_tool3</code>).</p>
<p>To find out more about how to change the default settings visit 'More options'.</p>
<h2 id="how-much-time-does-tool-3-take-to-run">How much time does Tool 3 take to run?</h2>
<p>A comparison of 2 strains can take as little as 12 minutes. The more strains you compare the longer it takes.</p>
<h2 id="what-are-the-different-folders-for">What are the different folders for?</h2>
<p>The Tool 3 file system contains the following folders:</p>
<table>
<thead>
<tr>
<th>Directory names</th>
<th align="left">Description of contents</th>
</tr>
</thead>
<tbody>
<tr>
<td>LIBRARY_query/</td>
<td align="left">The annotated files (<code>.gff</code>) for all sequenced isolates.</td>
</tr>
<tr>
<td>LIBRARY_reference/</td>
<td align="left">The annotated files (<code>.gff</code>) for the reference sequences.</td>
</tr>
<tr>
<td>input_tool3/</td>
<td align="left">The analysis will be run on ALL <code>.gff</code> files in this directory.</td>
</tr>
<tr>
<td>output_tool3/</td>
<td align="left">Your output data will be deposited here in subdirectories.</td>
</tr>
<tr>
<td>work/</td>
<td align="left">The Nextflow working directory.</td>
</tr>
</tbody>
</table>
<h2 id="what-documents-are-made-by-tool-3">What documents are made by Tool 3?</h2>
<p>Tool 3 provides the following outputs (listed alphabetically):</p>
<ul>
<li>accessory_binary_genes.fa               </li>
<li>accessory_binary_genes.fa.newick *</li>
<li>accessory_graph.dot             </li>
<li>accessory.header.embl          </li>
<li>accessory.tab</li>
<li>blast_identity_frequency.Rtab</li>
<li>clustered_proteins</li>
<li>core_accessory_graph.dot</li>
<li>core_accessory.header.embl</li>
<li>core_accessory.tab</li>
<li>core_alignment_header.embl</li>
<li>core_gene_alignment.aln</li>
<li>gene_presence_absence.csv</li>
<li>gene_presence_absence.Rtab</li>
<li>number_of_conserved_genes.Rtab</li>
<li>number_of_genes_in_pan_genome.Rtab</li>
<li>number_of_new_genes.Rtab</li>
<li>number_of_unique_genes.Rtab</li>
<li>pan_genome_reference.fa</li>
<li>summary_statistics.txt</li>
</ul>

<p>(*) output is only made if more than 2 strains are compared.</p>
<p>The outputs you will probably use the most are:</p>
<ul>
<li> a text document summarising the total number of core genes shared between isolates (summary_statistics.txt),</li>
</ul>

<p><img alt="Screenshot" src="img/ss_tool3_summary.png" /></p>
<ul>
<li> a spreadsheet showing presence/absence for every detected gene within your query isolates (gene_presence_absence.csv),  </li>
</ul>

<p><img alt="Screenshot" src="img/ss_tool3_presence_absence.png" /></p>
<ul>
<li> two files that can be viewed as phylogenetic trees (.newick). One that generates a tree based on an alignment of the core genes. And a second, created based on the binary presence and absence of accessory genes.  </li>
</ul>

<h2 id="how-do-you-view-the-output-files">How do you view the output files?</h2>
<p>You can open the files in a spreadsheet (e.g. MS Excel), a text editor (e.g. MS Word), or <a href="https://www.megasoftware.net/">Mega</a>/<a href="https://itol.embl.de/">iTOL</a>. </p>
<p>When opening the .csv file note it contains comma separated values (not tab separated).</p>
<h2 id="can-i-change-the-settings-of-tool-3">Can I change the settings of Tool 3?</h2>
<p>Yes, you can modify the two different settings: <code>identity</code> and <code>core</code> Using the default settings a gene must be present in 99% of the strains to be counted as a core gene, and have a minimum percentage identity (for blastp) of 95%. These can be altered when you set Tool 3 to run. </p>
<p>For all runs, the settings are recorded in the name of the newly created output folder, e.g. the results of a run named <code>dylan</code> using default settings will be stored in this folder: <code>output_dylan-i95-core99/</code>.</p>
<h2 id="where-can-i-find-further-information-about-tool-3">Where can I find further information about Tool 3?</h2>
<p>Tool 3 uses two software tools named roary and fast tree. Roary information page at <a href="http://sanger-pathogens.github.io/Roary/">Sanger Pathogens</a>
The FastTree information page can be found at <a href="http://www.microbesonline.org/fasttree/">Microbes Online</a>.</p>
              
            </div>
          </div>
          <footer>
  
    <div class="rst-footer-buttons" role="navigation" aria-label="footer navigation">
      
        <a href="glossary_3.html" class="btn btn-neutral float-right" title="Glossary">Next <span class="icon icon-circle-arrow-right"></span></a>
      
      
        <a href="unloading_3.html" class="btn btn-neutral" title="Downloading outputs"><span class="icon icon-circle-arrow-left"></span> Previous</a>
      
    </div>
  

  <hr/>

  <div role="contentinfo">
    <!-- Copyright etc -->
    
  </div>

  Built with <a href="https://www.mkdocs.org/">MkDocs</a> using a <a href="https://github.com/snide/sphinx_rtd_theme">theme</a> provided by <a href="https://readthedocs.org">Read the Docs</a>.
</footer>
      
        </div>
      </div>

    </section>

  </div>

  <div class="rst-versions" role="note" aria-label="versions">
  <span class="rst-current-version" data-toggle="rst-current-version">
    
        <span>
          <a href="https://github.com/volac-genomics/Tool3_docs/" class="fa fa-github" style="color: #fcfcfc"> GitHub</a>
        </span>
    
    
      <span><a href="unloading_3.html" style="color: #fcfcfc">&laquo; Previous</a></span>
    
    
      <span><a href="glossary_3.html" style="color: #fcfcfc">Next &raquo;</a></span>
    
  </span>
</div>
    <script>var base_url = '.';</script>
    <script src="js/theme_extra.js" defer></script>
    <script src="js/theme.js" defer></script>
      <script src="search/main.js" defer></script>
    <script defer>
        window.onload = function () {
            SphinxRtdTheme.Navigation.enable(true);
        };
    </script>

</body>
</html>
